eng/helix/Helix.proj

```diff
@@ -113,6 +113,14 @@
     <HelixTargetQueue Include="@(HelixAvailableTargetQueue)" />
   </ItemGroup>
 
+  <!-- Additional package feeds -->
+  <ItemGroup>
+    <AdditionalDotNetPackageFeed Include="https://dotnetbuilds.blob.core.windows.net/internal"
+                                 Condition="'$(SYSTEM_TEAMPROJECT)' == 'internal'">
+      <SasToken>$([System.Environment]::GetEnvironmentVariable('DotNetBuildsInternalContainerReadToken'))</SasToken>
```

Comment:
I was under the impression that enironment variables are automatically mapped to properties. Is this not the case for the token?

Reply:
This is how ASP.NET does it (I basically copied their solution). I can make a follow up change that switches to using the property if that works.

Reply:
You were right, it needs to be a property. ASP.NET must be setting it via the environment block as well somewhere. Fixed in https://github.com/dotnet/dotnet-monitor/pull/5262